test/test_view_ops.py

```diff
@@ -1701,20 +1702,20 @@ def test_tensor_split_errors(self, device):
 
     def test_resize_all_dtypes_and_devices(self, device):
         shape = (2, 2)
-        for dt in get_all_dtypes():
+        for dt in list(all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool)):
```

Comment:
What happens if these aren't converted to lists?

Reply:
Yeah, it works fine. We don't need to declare the list explicitly. I missed that. Thanks for the ping :)